### PR TITLE
ReportByLine for report with a context line with specific line

### PR DESCRIPTION
**Pull request by i582**
Solve #585

### CODE REVIEW

File: src/linter/root.go [hunk 1]
```diff
@@ -484,6 +484,76 @@ func (d *RootWalker) Report(n node.Node, level int, checkName, msg string, args
 	}
 }
 
+// ReportByLine registers a single report message about some found problem in lineIndex code line.
+func (d *RootWalker) ReportByLine(lineIndex int, level int, checkName, msg string, args ...interface{}) {
```
Comment (quasilyte) [inline src/linter/root.go:488]: Can we make a generalized report method that can implement major logic so we don't duplicate the code that manipulates that hash computations, etc?

File: src/linter/root.go [hunk 2]
```diff
@@ -666,8 +624,9 @@ func (d *RootWalker) handleComment(c freefloating.String) {
 			}
 			d.linterDisabled = canDisable
 			if !canDisable {
-				needleLine := ln.Line() + c.Position.StartLine - 2
+				needleLine := ln.Line() + c.Position.StartLine - 1
 				d.ReportByLine(needleLine, LevelInformation, "linterError", "You are not allowed to disable linter")
+				d.ReportByLine(19, LevelInformation, "linterError", "You are not allowed to disable linter")
```
Comment (quasilyte) [inline src/linter/root.go]: Probably a debug leftover?

Comment (i582) [reply src/linter/root.go]: already removed

File: src/linttest/testdata/output-test/output_test.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+/** @linter disable */
```
Comment (quasilyte) [inline src/linttest/testdata/output-test/output_test.php:3]: Nice job with adding a test for all kinds of phpdoc comments formatting.
:+1:

**Pull request merged**